@Test public void should_fail_since_actual_is_not_zero(){
  assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> numbers.assertIsNotZero(someInfo(),BigInteger.ZERO)).withMessage(format("%nExpecting:%n <0>%nnot to be equal to:%n <0>%n"));
}
